command/pullrequest/pull_request.go

```diff
@@ -241,6 +260,84 @@ func (c command) getApproveReactions(approvers []string) reactionMap {
 	return reactions
 }
 
+func getPRLinkMessage(prw *pullRequestWatch) string {
+	if len(prw.PullRequest.Link) > 0 {
+		return fmt.Sprintf("\nYou can check it here: %s", prw.PullRequest.Link)
+	}
+	return ""
+}
+
+func (c command) getBuildStatusIcon(pr pullRequest) string {
+	switch pr.BuildStatus {
+	case buildStatusRunning:
+		return fmt.Sprintf(":%s:", c.cfg.Reactions.BuildRunning)
+	case buildStatusSuccess:
+		return fmt.Sprintf(":%s:", c.cfg.Reactions.BuildSuccess)
+	case buildStatusFailed:
+		return fmt.Sprintf(":%s:", c.cfg.Reactions.BuildFailed)
+	case buildStatusUnknown:
+	}
+	return ""
+}
+
+func (c command) notifyBuildStatus(prw *pullRequestWatch) {
+	if len(prw.PullRequest.Author) == 0 {
+		// no author to notify
+		return
+	}
+
+	if prw.PullRequest.BuildStatus == prw.LastBuildStatus {
+		// build status didn't change
+		return
+	}
+
+	prw.LastBuildStatus = prw.PullRequest.BuildStatus
+
+	if c.cfg.Notifications.BuildStatusInProgress && prw.LastBuildStatus == buildStatusRunning {
+		c.sendPrivateMessage(prw.PullRequest.Author, "PR '%s'\nBuild Status: Started %s%s", prw.PullRequest.Name, c.getBuildStatusIcon(prw.PullRequest), getPRLinkMessage(prw))
+	}
+	if c.cfg.Notifications.BuildStatusSuccess && prw.PullRequest.BuildStatus == buildStatusSuccess {
+		c.sendPrivateMessage(prw.PullRequest.Author, "PR '%s'\nBuild Status: Success %s%s", prw.PullRequest.Name, c.getBuildStatusIcon(prw.PullRequest), getPRLinkMessage(prw))
+	}
+	if c.cfg.Notifications.BuildStatusFailed && prw.PullRequest.BuildStatus == buildStatusFailed {
+		c.sendPrivateMessage(prw.PullRequest.Author, "PR '%s'\nBuild Status: Failed %s%s", prw.PullRequest.Name, c.getBuildStatusIcon(prw.PullRequest), getPRLinkMessage(prw))
+	}
+}
+
+func (c command) notifyPullRequestStatus(prw *pullRequestWatch) {
+	if prw.DidNotifyMergeable {
+		return
+	}
+
+	if len(prw.PullRequest.Author) == 0 {
+		// no author to notify
+		return
+	}
+
+	if len(prw.PullRequest.Approvers) == 0 {
+		// prw isn't approved
+		return
+	}
+
+	if prw.PullRequest.BuildStatus != buildStatusSuccess {
+		// has still running or failed builds
+		return
+	}
+
+	prw.DidNotifyMergeable = true
+
+	c.sendPrivateMessage(prw.PullRequest.Author, "Your PR '%s' is ready to merge!%s", prw.PullRequest.Name, getPRLinkMessage(prw))
+}
+
+func (c command) sendPrivateMessage(username string, format string, parameter ...any) {
+	pmChannel := username
+	if !strings.HasPrefix(username, "@") {
+		pmChannel = fmt.Sprintf("@%s", username)
+	}
+	message := fmt.Sprintf(format, parameter...)
+	c.SendToUser(pmChannel, message)
```

Comment:
SendToUser() doesn't need the `@` prefix, it gets trimmed anyway in `identifier = strings.TrimPrefix(identifier, "@")`